<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportITunes</class>
 <widget class="QWizard" name="DialogImportITunes">
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Import from iTunes</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::AeroStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::HelpButtonOnRight|QWizard::NoBackButtonOnStartPage</set>
  </property>
  <widget class="QWizardPage" name="pageFile">
   <property name="title">
    <string>Library localisation</string>
   </property>
   <property name="subTitle">
    <string>Select the file which contains the iTunes library.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblFileName">
      <property name="text">
       <string>Library:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editFileName"/>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="btnFileName">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="pageItems">
   <property name="title">
    <string>Items to import</string>
   </property>
   <property name="subTitle">
    <string>Select items you want to import in your library.</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
